client/src/shared/components/Card/LearningPathCourseCard/index.tsx

```diff
@@ -1,23 +1,34 @@
 import Image from 'next/image';
+import Link from 'next/link';
 import React from 'react';
 
 interface Props {
-  imgPath: string;
+  courseId: number;
   courseTitle: string;
+  imgPath: string;
   lessonsCount: number;
```

Comment:
Since the card is for course, please pass the course itself instead, the less prop the better, and less confusing by other devs